api_tests/preprints/views/test_preprint_detail.py

```diff
@@ -127,6 +127,16 @@ def test_published_preprint_doi_link_returned_after_datacite_request(self, app,
         assert res.json['data']['links']['preprint_doi'] == 'https://dx.doi.org/{}'.format(expected_doi)
         assert res.json['data']['attributes']['preprint_doi_created'] is not None
 
+    def test_preprint_embed_identifiers(self, app, user, preprint, url):
+        expected_doi = EZID_FORMAT.format(namespace=DOI_NAMESPACE, guid=preprint._id).replace('doi:', '')
```

Comment:
The first three lines of this test can be removed. You don't actually need to fake the DOI as the identifiers relationship will be serialized whether the preprint has identifiers or not.